### PR TITLE
Arrows

### PR DESCRIPTION
**Pull request by TheDiggie**

### CODE REVIEW

**Comment by skittles1** [comment]
Really think we should be using separate colors (even if it's just a slightly lighter shade of green) and different quiver design for these. Acid arrows could use a green fire/ice quiver, and these changes will allow players to easily identify them in inventory or when they're being fired.

**Comment by TheDiggie** [comment]
added:
acid arrows to earth golems
fire arrows to fire golems
ice arrows to ice golems
removed wands from fire golems

**Comment by GarOfMeridian** [comment]
These are pretty significant changes to WC. I think we talked about this back in the day, but giving bows access to so many elements is kinda huge. The elements a given school has access to give the school character and certain roles, and I'm pretty sure elements will be important in upcoming systems like Narwhal's gear crafting. It's cool that these arrows exist, but actually making them drops in the game suddenly gives bow access to every single element type. 

So it's like Qor specializes in Acid, Faren specializes in fire/ice/shock, but somehow plain old arrows can do all of them? I can already be sure, too, that fire arrows will be superior to fireball (because fireball sucks) - so a school of magic focused around dealing damage with the elements is worse at dealing ranged fire than a weapon school ?

File: resource/graphics/diggie/makefile [hunk 1]
```diff
@@ -7,8 +7,9 @@ OUTDIR = .
 .SUFFIXES: .bbg
 
 BGFS = neeleme.bgf neelemeX.bgf iunholyt.bgf xeoneru.bgf xeoneruX.bgf \
+
```
Comment (skittles1) [inline resource/graphics/diggie/makefile]: This added blank line is causing the build to fail.

**Pull request merged**